doc/api/stream.md

```diff
@@ -1508,6 +1508,9 @@ the callback and passing the error as the first argument. This will cause an
 on how the stream is being used.  Using the callback ensures consistent and
 predictable handling of errors.
 
+*Note*: If a Readable stream pipes into a Writable stream when Writable emits an
+error, Readable stream will be unpiped.
```

Comment:
Nit: add `the` before `Readable stream`.

Reply:
Done.